ruby.c

```diff
@@ -381,6 +385,9 @@ usage(const char *name, int help, int highlight, int columns)
         M("frozen-string-literal", "", "Freeze all string literals (default: disabled)."),
 #if USE_YJIT
         M("yjit",                  "", "In-process JIT compiler (default: disabled)."),
+#endif
```

Comment:
elif here too, as above?

Reply:
Actually, the elif above should be changed to if, since we'll have YJIT+ZJIT builds.

Reply:
oh, ok